### PR TITLE
Incorporation of datastore library

### PR DESCRIPTION
**Pull request by MathewLG**

### CODE REVIEW

File: portfolio/src/main/java/UploadServlet.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
+// Copyright 2019 Google LLC
```
Comment (tucosh) [inline portfolio/src/main/java/UploadServlet.java:1]: It isn't strictly necessary, but note that in any real-world scenario all classes should be in a package, whose name by convention identifies the owning organisation. So the first line of the file might be for example

package com.mathewlg.sps;

.. and the file itself would need to be in directory portfolio/src/main/java/com/mathewlg.sps

Idem for the other servlet

File: portfolio/src/main/java/UploadListServlet.java [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+@WebServlet("/list-uploads")
+public class UploadListServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // List all of the uploaded files.
+    String projectId = "mlopezgarcia-sps-spring21";
+    String bucketName = "mlopezgarcia-sps-spring21.appspot.com";
+    Storage storage = StorageOptions.newBuilder().setProjectId(projectId).build().getService();
+    Bucket bucket = storage.get(bucketName);
```
Comment (tucosh) [inline portfolio/src/main/java/UploadListServlet.java:22]: Does this work for you?

When I run this code, I get an exception here at line 22:
com.google.cloud.storage.StorageException: java.lang.IllegalArgumentException: no JSON input found
        at com.google.cloud.storage.StorageException.translateAndThrow(StorageException.java:81)
        at com.google.cloud.storage.StorageImpl.get(StorageImpl.java:296)
        at UploadListServlet.doGet(UploadListServlet.java:22)

Might be a permissions problem? I am able to brows the items in your bucket with my browser, using url
https://console.cloud.google.com/storage/browser/mlopezgarcia-sps-spring21.appspot.com

File: portfolio/src/main/java/UploadServlet.java (hunk 1 above)
Comment (MathewLG) [reply portfolio/src/main/java/UploadServlet.java]: Done

File: portfolio/src/main/java/UploadListServlet.java (hunk 2 above)
Comment (MathewLG) [reply portfolio/src/main/java/UploadListServlet.java]: Yes, it works, might have something to do with permissions

Comment (tucosh) [reply portfolio/src/main/java/UploadListServlet.java]: Ya I'm not sure whether it's supposed to work when running locally, let me know when it's deployed to app server and I'll try again.

**Review by tucosh — APPROVED** [review]
LGTM

**Pull request merged**